server/views/errors/authError.njk

```diff
@@ -4,21 +4,22 @@
     {% block content %}
         <div class="govuk-grid-row">
             <div class="govuk-grid-column-two-thirds">
-                <h1 class="govuk-heading-l">Sorry, you are not authorised to access this page</h1>
-
-                {% if message or accessErrors %}
-                    <p class="govuk-body">The following issues have been identified:</p>
-                    {% if message %}
-                        <p class="govuk-body"><strong>{{ message }}</strong></p>
-                    {% endif %}
+                {% if not userHeader %}
+                    <h1 class="govuk-heading-l">Sorry, you are not authorised to access this page</h1>
+                {% endif %}
+                {% if userHeader %}
+                    <h1 class="govuk-heading-l">{{ userHeader }}</h1>
+                {% endif %}
+                {% if userMessage %}
+                    <p class="govuk-body">{{ userMessage | safe }}</p>
+                {% endif %}
                     {% if accessErrors %}
```

Comment:
_nit, not a blocker_ Indentation seems to be off -- do we have a linter to catch these?